x-pack/legacy/plugins/uptime/public/components/functional/location_map/embeddables/map_config.ts

```diff
@@ -5,7 +5,7 @@
  */
 
 import lowPolyLayerFeatures from './low_poly_layer.json';
-import { LocationPoint } from './embedded_map';
+import { LocationPoint } from './embedded_map.js';
```

Comment:
Is this a typo?

Reply:
My editor automatically did that.. I don't know what the typical convention is but I've seen it elsewhere in Kibana too.